.changeset/plain-rules-kneel.md

```
@@ -0,0 +1,12 @@
+---
+"@daiso-tech/core": minor
+---
+
+- Breaking Changes
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)